packages/foundry/script/DeployHelpers.s.sol

```diff
@@ -15,20 +15,37 @@ contract ScaffoldETHDeploy is Script {
   string root;
   string path;
   Deployment[] public deployments;
+  uint256 constant ANVIL_FIRST_PK =
+    0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80;
+  address constant ANVIL_FIRST_ACCOUNT =
+    0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266;
```

Comment:
Maye instead of using anvils first account to send funds maybe using anvil's last account make more sense? 

Because in SE-2 frontend on localhost anvil first account behaves as faucet to send fund to burner wallet. 

So I think maybe we should use last account just to be safe